tools/text_normalization/normalize.py

```diff
@@ -41,6 +61,24 @@
     tag_verbatim,
 ]
 
+verbalizers = {
+    TagType.CARDINAL: [expand_cardinal, expand_roman],
```

Comment:
Could you please change `taggers` to TAGGERS and verbalizers -> VERBILIZERS?
Maybe define them in the tagger.py and just import here?

Reply:
there would be an import loop, but I will Capitalize them